scripts/plot_allocations.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
```

Comment:
Even though this is a script, it would be better in the `tools` sub-directory.

Reply:
Also, once you move it, please update the CMakeLists.txt to make sure the script is installed.

Reply:
👍